rfd/0009-viewer-app.md

```diff
@@ -0,0 +1,76 @@
+---
+authors: Mathew (wrm932@gmail.com)
+state: draft
+---
+
+# Required Approvers
+* Engineering @pierrebeaucamp && @jimbishopp && @michellescripts && @mcbattirola
+
+# RFD 0009 - RFDs
+
+## What
+
+Create a directory viewer similar to Github and Google drive. 
+
+
+## Why
+
+A pratical exam that tests the potential applicant on their fullstack software knowledge. 
+
+## Details
```

Comment:
What technologies are you planning to use? Languages, frameworks, tools etc.

Reply:
Can you add a section on your testing strategy, which tests you plan on implementing, and why?

Reply:
@mcbattirola Good catch I'll add the tech stack! 

@michellescripts That's a good call out. I'll add the testing strategy.